Documentation/architecture/net-platform-standard.md

```diff
@@ -1,5 +1,7 @@
 ﻿# .NET Platform Standard
 
+> The .NET Platform Standard concept has been replaced by the .NET Standard Library. Check out the [.NET Standard Library](https://docs.microsoft.com/dotnet/articles/standard/library) topic in the [.NET Documentation](https://docs.microsoft.com/dotnet). It provides a good first view into .NET Standard Library.
```

Comment:
If .NET Platform Standard has been deprecated in favor of .NET Standard Library, should this whole page just be deleted?